packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/routing.js

```diff
@@ -124,15 +148,19 @@ export default /* @ngInject */ ($stateProvider) => {
           .then(({ data }) => data)
           .catch(() => ({})),
 
-      applyPolicy: /* @ngInject */ ($http, serviceName, volumeId) => (
-        snapshotPolicyID,
-      ) =>
+      /* @ngInject */
+      applyPolicy: ($http, serviceName, volumeId) => (snapshotPolicyID) =>
         $http.put(
           `/storage/netapp/${serviceName}/share/${volumeId}/snapshotPolicy`,
           {
             snapshotPolicyID,
           },
         ),
     },
+    atInternet: {
+      rename: `${SNAPSHOT_TRACKING_PREFIX}netapp::listing::snapshots`,
+      ...SNAPSHOT_LISTING_TRACKING_CONTEXT,
+      page_category: 'listing',
+    },
```

Comment:
Did you verified if page_theme and page_catagory is tracked here ? IMO its not and we need to ignore this and trackPage in controller